browser_history/cli.py

```diff
@@ -16,44 +18,78 @@ def make_parser():
-                                      epilog='''
+                                            (almost) any browser on (almost) any platform""",
+        epilog="""
                                             Checkout the GitHub repo https://github.com/pesos/browser-history
-                                            if you have any issues or want to help contribute''')
-
-    parser_.add_argument('-b', '--browser',
-                         default='all',
-                         help=f'''
-                                browser to retrieve history from. Should be one of all, {AVAILABLE_BROWSERS}.
-                                Default is all (gets history from all browsers).''')
-
-    parser_.add_argument('-f', '--format',
-                         default="csv",
-                         help=f'''
+                                            if you have any issues or want to help contribute""",
+    )
+
+    parser_.add_argument(
+        "-t",
+        "--type",
+        default="history",
+        help=f"""
+                                argument to decide whether to retrieve history or bookmarks.
+                                Should be one of all, {AVAILABLE_TYPES}.
+                                Default is history
+                                .""",
+    )
+    parser_.add_argument(
+        "-b",
+        "--browser",
+        default="all",
+        help=f"""
+                                browser to retrieve history or bookmarks from. Should be one of all, {AVAILABLE_BROWSERS}.
+                                Default is all (gets history or bookmarks from all browsers).""",
+    )
+
+    parser_.add_argument(
+        "-f",
+        "--format",
+        default="csv",
+        help=f"""
                                 Format to be used in output. Should be one of {AVAILABLE_FORMATS}.
-                                Default is csv''')
+                                Default is csv""",
+    )
+
+    parser_.add_argument(
+        "-o",
+        "--output",
+        default=None,
+        help="""
+                                File/Folder where history output and/or bookmark output is to be written. 
+                                If not provided, standard output is used.""",
+    )
 
-    parser_.add_argument('-o', '--output',
-                         default=None,
-                         help='''
-                                File where output is to be written. 
-                                If not provided standard output is used.''')
     return parser_
 
+
 parser = make_parser()
 
+
 def main():
     """Entrypoint to the command-line interface (CLI) of browser-history.
 
     It parses arguments from sys.argv and performs the appropriate actions.
     """
     args = parser.parse_args()
-
-    if args.browser == 'all':
-        outputs = get_history()
-
+    assert args.type in [
+        "history",
+        "bookmarks",
+        "all",
+    ], f"Type should be one of all, {AVAILABLE_TYPES}"
```

Comment:
this can be a tuple instead of a list

Reply:
Alternatively, you could instead use the keys of `fetch_map` rather than hard-coding `history`, `bookmarks` here.